plugins/modules/apache2_mod_proxy.py

```diff
@@ -246,11 +246,11 @@ class BalancerMember(object):
 
     def __init__(self, management_url, balancer_url, module):
         self.host = regexp_extraction(management_url, EXPRESSION, 4)
-        self.management_url = str(management_url)
+        self.management_url = management_url
         self.protocol = regexp_extraction(management_url, EXPRESSION, 3)
         self.port = regexp_extraction(management_url, EXPRESSION, 5)
         self.path = regexp_extraction(management_url, EXPRESSION, 6)
```

Comment:
Unrelated to this PR, but the above is pretty inefficient: it matches `management_url` with the same regex `EXPRESSION` four times, every time extracting a different group. Matching it once and then extracting all groups (if there is a match) would be more efficient.

Reply:
yup, this and other PRs before (maybe after) were about reducing the changes from #9457 . After that one gets merged, it will hopefully be easier to make improvements.